pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_creating-a-product.adoc

```diff
@@ -1,28 +1,28 @@
 [id="creating-a-product_{context}"]
-= Creating a product
+= Creating a product in Pantheon
 
 [role="_abstract"]
 A product must exist in {ProductName} before you can assign a product name to the metadata of a module or assembly. If the product doesn't already exist, you need to create one.
 
 .Prerequisites
 
-* You are logged in.
-* You have Publisher permissions.
+* You are logged in to {ProductShortName}.
+* You are assigned the publisher role.
 
 .Procedure
 
 . Navigate to menu:Products[New Product].
 
-. Enter the following information:
-  Product Name:: The text that appears in {ProductShortName}, the document preview, and in user-facing content.
-  Product UrlFragment:: This string appears in the user-facing URL for any content that belongs to this product and version. A URL fragment can include letters, numbers, hyphens, periods and underscores.
-  Product Description (optional):: A description of the product.
+. Complete the following information:
+  Product Name:: Enter the name for your product. The product name displays in {ProductShortName}, the document preview, and in user-facing content.
```

Comment:
General comment: Don't use "displays" in this way. I see it a lot as an attempt to avoid the passive tense "is displayed", but it should be either "is displayed" (I know it's passive, but it's clear) or "appears". Here's a passage from page 233 of the latest IBM style guide:

For group 

> elements such as windows, menus, lists, and group boxes, you can use the verb form "is
> displayed" to describe the presence of the element on the screen. For windows and wizards, you can
> use "opens" to describe the immediate result of a user action. Do not use "appears" or the active verb
> "displays".

And page 315:

> 
> display verb
> Use only as a transitive verb. For example, write "the system displays a message" or "the message is
> displayed" (not "the message displays").

Reply:
Thanks. I'd rather go with "is displayed". The latest IBM style guide doesn't mention it but in the previous versions they discourage from using "appear". "Do not use appear unless there is no better alternative. Possible alternatives include "is displayed" and "opens". "Appear" implies magic or illusion..." 
Unless you really want to avoid passive tense.

Reply:
+1